showcase/src/app/welcome/pages/home/_home-page.component.scss

```diff
@@ -1,94 +1,168 @@
-//set the header image
-.about-background {
+.centered-title {
+  text-align: center;
+  font-size: 17px;
+  line-height: 30px;
+  font-weight: 500;
+}
+
+.item-alignment {
   display: flex;
+  flex-direction: column;
+  justify-content: center;
   align-items: center;
-  
-  margin-bottom: 20px;
-  height: 100px;
+  box-sizing: border-box;
+  margin: 16px auto;
+}
+
+.doc-subtitle {
+  margin-left: 25px;
+}
 
-  background-color: mat-color($primary-palette, 900);
-  background-image: url(/assets/stark-core/logo/light/stark_logo_light.svg);
+.stark-logo{
+  background-image: url(/assets/stark-core/logo/dark/stark_logo_dark.svg);
+  width: 250px;
+  height: 100px;
+  background-size: contain;
   background-repeat: no-repeat;
-  background-size: 250px 100px;
-  background-position: center;
+  display: flex;
+  margin: auto;
 }
 
-.light-font {
-  font-weight: normal;
+.core-image {
+  mask-image: url("/assets/images/logo/atom.svg");
+  mask-repeat: no-repeat;
+  mask-size: contain;
+  background-color: mat-color($accent-palette, 500);
+  height: 80px;
+  width: 80px;
+  min-width: 80px;
+  max-width: 80px;
 }
 
-.bottom-margin {
-  margin-bottom: 20px;
+.about-image {
+  mask-image: url("/assets/images/logo/wall.svg");
+  mask-repeat: no-repeat;
+  mask-size: contain;
+  background-color: mat-color($primary-palette, 500);
+  height: 80px;
+  width: 80px;
+  min-width: 80px;
+  max-width: 80px;
 }
 
-//icon with a link to github
-.github-link .github-icon {
-  height: 60px;
-  width: 60px;
-  position: relative;
-  float: right;
+.ui-image {
+  mask-image: url("/assets/images/logo/television-guide.svg");
+  mask-repeat: no-repeat;
+  mask-size: contain;
+  background-color: mat-color($success-palette, 500);
+  height: 80px;
+  width: 80px;
+  min-width: 80px;
+  max-width: 80px;
+}
 
-  svg {
-    fill: #000;
-  }
+.showcase-image {
+  mask-image: url("/assets/images/logo/theater.svg");
+  mask-repeat: no-repeat;
+  mask-size: contain;
+  background-color: mat-color($warning-palette, 900);
+  height: 80px;
+  width: 80px;
+  min-width: 80px;
+  max-width: 80px;
 }
 
-.information-title {
-  font-size: 21px;
-  color: mat-color($grey-palette, 900);
-  padding-bottom: 0;
-  display: inline-flex;
-  vertical-align: middle;
-  margin: 10px 0 10px 0;
+.page-content {
+  margin: 0;
 }
 
-@media #{$desktop-query} {
-  .about-title {
-    font-size: 55px;
-    margin-left: 25px;
-  }
+.doc-description {
+  display: flex;
+  margin-left: 25px;
+  margin-right: 25px;
 
-  .information-title {
-    font-size: 30px;
+  p {
+	display: flex;
   }
+}
 
-  .about-background {
-    height: 150px;
-    background-size: 250px 75px;
-  }
+.custom-button {
```

Comment:
Can you refactor the styles for `.custom-buttom`, `.custom-buttons` and `.buttons-row` so that it reflects the structure that they have in the HTML? This way I think you can clean up some unnecessary code and make it readable ;)